### PR TITLE
Security Medic SOP and department changes

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

This basically aims to tackle the problem of the security medic taking the role of the paramedic. 

They now report to the Chief Medical Officer while in medbay and have very bold warnings enforcing SOP onto them.

"As a Security Medic your primary focus is to keep prisoners and officers alive. This means you should not be focusing on civilian type medical duties, leave that to the paramedic and medical. Do not make a mini-medical center within security, always take people that need in-depth medical attention to medbay. If you are in medbay, you report to the CMO!"

This also vetlocks the role as to try and root out bad faith actors. 

## Changelog

:cl:
add: The Security Medic has been given updated SOP and now reports to the CMO while in medbay. It is also now veteran locked.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm [hunk 1]
```diff
@@ -43,6 +43,13 @@
 	rpg_title = "Battle Cleric"
 	job_flags = JOB_ANNOUNCE_ARRIVAL | JOB_CREW_MANIFEST | JOB_EQUIP_RANK | JOB_CREW_MEMBER | JOB_NEW_PLAYER_JOINABLE | JOB_REOPEN_ON_ROUNDSTART_LOSS | JOB_ASSIGN_QUIRKS | JOB_CAN_BE_INTERN
 
+	veteran_only = TRUE
+
+/datum/job/security_medic/after_spawn(mob/living/spawned, client/player_client)
+	. = ..()
+	to_chat(player_client, span_doyourjobidiot("As a Security Medic your primary focus is to keep prisoners and officers alive. This means you should not be focusing on civilian type medical duties, leave that to the paramedic and medical. Do not make a mini-medical center within security, always take people that need in-depth medical attention to medbay. If you are in medbay, you report to the CMO!"))
```
Comment (Zonespace27) [inline modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm]: isn't there a config file for doing exactly this?

Comment (Gandalf2k15) [reply modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm]: whewre tho

Comment (Zonespace27) [reply modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm]: `policy.json`

**Comment by Nerev4r** [comment]
What need does this serve that Paramedic does not?

File: modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm (hunk 1 above)
Comment (Gandalf2k15) [reply modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm]: done

**Comment by nikothedude** [comment]
security really doesn't need a dedicated healer, all secmed is gonna do is make sec even more secluded/isolated from the rest of the station even more than they already are since they dont even need to go to medbay

and yes i know things like surgery go to medbay - but why are we even encouraging security to avoid medical so directly at _all?_ the prescense of this job and it's policy directly imply that the purpose of secmed existing is to allow security to load what would normally be a job for medical back onto _themselves_
does security really get injured so much and in such inconvenient positions that it's bad for the game for them to haev to go to medical? or allow a paramedic to do the healing - can they not trust paramedics? because the last time i saw, the amount of hell shifts we have had that precludes security from getting hasty healing has been absolutely zero - and it would be a problem if it happened a _lot_. which it dont

i mean if you can rationalize that, you can defeat most arguments against secmed revolving around sec's isolation frankly

**Comment by Nerev4r** [comment]
Let me rephrase that.
What role does Security Medic have in the current station ecosystem? 
Within the server as it is right now, we already have far less chaos and general murder/injury than we had months prior. Guncargo's in a nerfed state, assops doesn't really roll, and the current Security numbers (augmented by what _is_ there in Cargo and the Armory are more than capable of handing whatever's thrown at them. We regularly have Medical players ingame and on the Discord complaining that short of blobs, we do **not** have mass-casualty events necessitating more than, really, a Paramedic, maybe a cyborg, and a few Doctors.

Even then, those are all short on work. Security, let alone the Prisoners, have no current big gameplay niche for an in-house Doctor, especially given that this role has been gone for several months now. At best, we could give COs a medkit or two for prisoners, but I don't often hear of Medical being dispatched to the Brig anyway (which they can be.) _Even_ then, if either camp needed an in-house Doctor, it would likely be best for it to not be a role that's both metaprotected and has the ability to arm up and function policy-wise as a Security unit. 

Lastly, as we had issues with the last time this was a role in the game, Security _does and did_ make up a bulk of the actual injuries in a round. Yes, we have 'always take people that need in-depth medical attention to medbay' in this new PR, but field stabilization and basic injury-tending is, literally, the job description of the Paramedic as we already had in Medical policy. Medical policy that this role does not fit into, and is not integrated with. Robbing Medical of that primary group of injured players, **while** reintroducing a role that isolates Security from the wider station ecosystem, is not good.

**Comment by nikothedude** [comment]
also i would just like to predict something that might be said and make a counteragument ahead of time

just because secmed isnt being added in this pr doesnt mean this pr doesnt imply it is intended to be added (policy change to a disabled role does imply this) - thats why people are acting like its being re-added

**Comment by dawsonkeyes** [comment]
What happened to secmed being gone for good? It feels exceptionally redundant, they do everything a paramedic does except they're involved in security matters as well. Better yet, why vetlock it? What exactly can "bad faith actors" do as a gun-toting paramedic that they can't do as a paramedic, or better yet, an actual security officer?

Updating the SOP doesn't really change any of what they actually do -- everything listed is the job of a paramedic anyways.

**Comment by nikothedude** [comment]
also being vetlocked doesnt stop the implications of playing the role as-intended, which are still. not good

**Comment by Gandalf2k15** [comment]
> security really doesn't need a dedicated healer, all secmed is gonna do is make sec even more secluded/isolated from the rest of the station even more than they already are since they dont even need to go to medbay
> 
> and yes i know things like surgery go to medbay - but why are we even encouraging security to avoid medical so directly at _all?_ the prescense of this job and it's policy directly imply that the purpose of secmed existing is to allow security to load what would normally be a job for medical back onto _themselves_ does security really get injured so much and in such inconvenient positions that it's bad for the game for them to haev to go to medical? or allow a paramedic to do the healing - can they not trust paramedics? because the last time i saw, the amount of hell shifts we have had that precludes security from getting hasty healing has been absolutely zero - and it would be a problem if it happened a _lot_. which it dont
> 
> i mean if you can rationalize that, you can defeat most arguments against secmed revolving around sec's isolation frankly

I mean, a security medic can help maintain the effectiveness of the security team by providing quick and efficient medical care during emergencies when they would normally not be able to. This allows the security officers to return to their duties faster and helps to prevent a mass casualty event for the security team should a situation arise. And also, during chaotic situations, such as large-scale attacks or riots, it may be dangerous for security officers to venture into the medical bay due to ongoing threats, hell, it might not even be accessible. 

The point of a security medic is to aid on the field as stated in its inception. Paramedics shouldn't be attending gunfights, which, in practice, they don't. You also have the trust issue, Security officers may be more likely to trust a security medic, who is part of their team, over a paramedic or other medical staff. This can help ensure that injured officers receive the care they need, without hesitating due to concerns about potential sabotage or infiltration by antagonists.

I also don't see how it promotes isolation given the security medic also has to actually go to medical. Whom is it isolating? Security is still going to Medbay for stuff that can't just be treated on-site so they can you know, tend to the threat. Out of both the paramedic and the security officer, the more redundant one is the paramedic.  It also makes prisoners feel less isolated and gives them something to interact with more freely. You will not see a paramedic in security for no reason, so the chances of prisoner-secmed rp is much much higher. I also fail to see how the argument here, security officers already carry medkits to do the job of a paramedic, this just adds some roleplay to security.

The argument can also be made that how can the security medic "steal" the job of a paramedic when there are already 3 of them on-site? There isn't much to steal. It's a pretty silly argument. This role is mostly used for its roleplay value given it adds another layer of interaction for the prisoners and a lot of people enjoy that. I'm sorry but if one single person is enough to steal the content of another job, then that job is the one that lacks the content, especially if that is the primary argument against another job. And even then, the security medic doesn't go around healing people randomly.

Overall the job itself absolutely serves a purpose as stated above. It can be argued that we don't need 3 paramedics but hey, we still do. Also, this just changes the SOP and supervisors, and vetlocks it. The actual enabling is done serverside, which is not going to be done here.

**Comment by dawsonkeyes** [comment]
> The point of a security medic is to aid on the field as stated in its inception. Paramedics shouldn't be attending gunfights, which, in practice, they don't. You also have the trust issue, Security officers may be more likely to trust a security medic, who is part of their team, over a paramedic or other medical staff. This can help ensure that injured officers receive the care they need, without hesitating due to concerns about potential sabotage or infiltration by antagonists.

What exactly is wrong with this? Is the input of players that distrust paramedics in this theoretical situation not an inherently good thing, adding depth to roleplay and engagement among roles? Why should all of security's matters be so self-contained? What's wrong with paramedics attending gunfights, when it's their job to get people out of them when they're in critical condition, something that's become a regularity on the server already?

**Comment by nikothedude** [comment]
> I mean, a security medic can help maintain the effectiveness of the security team by providing quick and efficient medical care during emergencies when they would normally not be able to. This allows the security officers to return to their duties faster and helps to prevent a mass casualty event for the security team should a situation arise. And also, during chaotic situations, such as large-scale attacks or riots, it may be dangerous for security officers to venture into the medical bay due to ongoing threats, hell, it might not even be accessible.

but why is giving security better access to quick and efficient care _good for the game?_ especially again when you understand the fact taht merely having a secmed present will intrinsically remove some work for medical at all times because security is never ever going to show up for anything that doesn't require things like surgery 

plus, again - security really is not overwhelmed. like ever, here. i cant remember the lsat time it happened. and sometimes, they can be overwhelmed! this isnt intrinsically bad, as long as it doesnt happen constantly, which it doesnt. 

> You also have the trust issue, Security officers may be more likely to trust a security medic, who is part of their team, over a paramedic or other medical staff. This can help ensure that injured officers receive the care they need, without hesitating due to concerns about potential sabotage or infiltration by antagonists.

same athing as above! why is it so bad for secoffs to not trust the people treating them? let them hesitate.

i will say: your argument about secmed and prisoners is solid and i like it but i dont like the idea of secmed's purpose being to act as a dedicated paramed for sec.

**Comment by Gandalf2k15** [comment]
> > The point of a security medic is to aid on the field as stated in its inception. Paramedics shouldn't be attending gunfights, which, in practice, they don't. You also have the trust issue, Security officers may be more likely to trust a security medic, who is part of their team, over a paramedic or other medical staff. This can help ensure that injured officers receive the care they need, without hesitating due to concerns about potential sabotage or infiltration by antagonists.
> 
> What exactly is wrong with this? Is the input of players that distrust paramedics in this theoretical situation not an inherently good thing, adding depth to roleplay and engagement among roles? Why should all of security's matters be so self-contained? What's wrong with paramedics attending gunfights, when it's their job to get people out of them when they're in critical condition, something that's become a regularity on the server already?

It's not their job though, that would result in validhunting a lot of the time. At least with this you have a specific person that is easily identified as a member of security instead of having to worry about if the paramedic is going to syringe gun you.

**Comment by Nerev4r** [comment]
> The point of a security medic is to aid on the field as stated in its inception. Paramedics shouldn't be attending gunfights, which, in practice, they don't. You also have the trust issue, Security officers may be more likely to trust a security medic, who is part of their team, over a paramedic or other medical staff. This can help ensure that injured officers receive the care they need, without hesitating due to concerns about potential sabotage or infiltration by antagonists.

> I also don't see how it promotes isolation given the security medic also has to actually go to medical. Whom is it isolating? Security is still going to Medbay for stuff that can't just be treated on-site so they can you know, tend to the threat. Out of both the paramedic and the security officer, the more redundant one is the paramedic. It also makes prisoners feel less isolated and gives them something to interact with more freely. You will not see a paramedic in security for no reason, so the chances of prisoner-secmed rp is much much higher. I also fail to see how the argument here, security officers already carry medkits to do the job of a paramedic, this just adds some roleplay to security.

This, literally, contradicts itself. If Security Officers trust the Security Medic, but the Security Medic is going to Medical, doesn't this cancel itself out? Doesn't this not make any sense? And, yes, 'not trusting Medical' is part of the base gameplay. It's why we don't have a Cargo Medic, or Engineering Medic, or Service Medic. Everyone ends up on the OR table, or being injected on-site by whatever a Paramedic has, by design. This is the game.

**Comment by Gandalf2k15** [comment]
> > I mean, a security medic can help maintain the effectiveness of the security team by providing quick and efficient medical care during emergencies when they would normally not be able to. This allows the security officers to return to their duties faster and helps to prevent a mass casualty event for the security team should a situation arise. And also, during chaotic situations, such as large-scale attacks or riots, it may be dangerous for security officers to venture into the medical bay due to ongoing threats, hell, it might not even be accessible.
> 
> but why is giving security better access to quick and efficient care _good for the game?_ especially again when you understand the fact taht merely having a secmed present will intrinsically remove some work for medical at all times because security is never ever going to show up for anything that doesn't require things like surgery
> 
> plus, again - security really is not overwhelmed. like ever, here. i cant remember the lsat time it happened. and sometimes, they can be overwhelmed! this isnt intrinsically bad, as long as it doesnt happen constantly, which it doesnt.
> 
> > You also have the trust issue, Security officers may be more likely to trust a security medic, who is part of their team, over a paramedic or other medical staff. This can help ensure that injured officers receive the care they need, without hesitating due to concerns about potential sabotage or infiltration by antagonists.
> 
> same athing as above! why is it so bad for secoffs to not trust the people treating them? let them hesitate.
> 
> i will say: your argument about secmed and prisoners is solid and i like it but i dont like the idea of secmed's purpose being to act as a dedicated paramed for sec.

well, I've been told that the general consensus is that we want the station to "win". It's been echoed about, but the level of medical care that can be given by a secmed is nothing compared to an MD. But the primary purpose of this, providing people adhere to it, is to add more dynamic interaction for security instead of it always being the primary opposer. We would likely reduce the security officer count by one if we re-enabled it, this way we can in some ways give antags a better chance, but also make it so we can keep security in the game which might benefit us in the long run, if an officer can stay alive for longer, they'll be less likely to resent an antagonist killing them. It's a slight dig on the hate that security has for antags. But it also helps antags. It's a pretty dynamic role. One of the biggest and best reasons to have it though is the prisoner-sec RP it absolutely provides, and those that play the role properly will encounter this a lot. I see people wanting it often, and there have been several requests. I just don't see how the arguments against outweigh the arguments for.

**Comment by softcerv** [comment]
As I said on discord, I'd advise giving security medic policy the `doyourjobidiot` span notice that is given when using alternate job titles. While I don't think it will entirely prevent bad behavior, it will give people less of an excuse.

**Comment by YakumoChen** [comment]
Validing Paras is an admin issue. 

Sec not being able to trust the medic is a good thing, security being injured is the perfect time for antag medical to roleplay with a job that is typically anti-roleplay with any manner of antag shenanigans. Abducting an injured officer to an maint doctor's lair is exactly the kind of RP we need.

**Comment by OzzyMcPseudonym** [comment]
Secmed is the exact opposite of what we need as a server. All they do is tide the syringe gun that should be used by an orderly and take away content from medical. The server has been better ever since it was removed. I unironically almost never enjoyed any interactions with sec medics because of how the design of the job conflicts with the design of the game as a whole. The game is built around departments being loosely tied together through interdepartmental cooperation to an extent, with security being the department that lacks this cooperation comparative to the other departments. Sec medic takes a department that is, by its nature, more insular than other departments, and makes them even more isolated from everyone else by making a majority of cases where someone in sec needs healing simply no longer involve another department at all.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
I still don't believe this is needed on Skyrat in any way whatsoever. The only people that enjoyed Security Medics were the Security Medics, and Security.

IC conflict and being unable to trust the doctors 100% is _good_, and having a roundstart mindshielded doctor only brings problems.

I still maintain that if you want a Security Medic in a round, you should be deputizing a Paramedic, or a Medical Doctor for a Brig Physician.

But as it stands, I don't think it's beneficial to see Security Medics coming back to the server without an actual rework to make them into a sidegrade of an existing job, as an in-round kind of deal.

**Comment by tf-4** [comment]
what golden said
